viz/charts/linkerd-viz/templates/admin-policy.yaml

```diff
@@ -25,9 +25,27 @@ metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
 spec:
-  server:
+  targetRef:
+    group: policy.linkerd.io
+    kind: Server
     name: admin
-  client:
-    # for kubelet probes and prometheus scraping
-    unauthenticated: true
-
+  # Permits traffic from the kubelet. This could be restricted further and share
+  # a common network resource.
+  requiredAuthenticationRefs:
+  - group: policy.linkerd.io
+    kind: NetworkAuthentication
+    name: admin-unauthenticated
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: NetworkAuthentication
+metadata:
+  {{ include "partials.namespace" . }}
+  name: admin-unauthenticated
+  labels:
+    linkerd.io/extension: viz
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+spec:
+  networks:
+  - cidr: "0.0.0.0/0"
+  - cidr: "::/0"
```

Comment:
Ideally, I think we'd provide cluster-wide defaults in the linkerd namespace, via a `kubelet-nets` `NetworkAuthentication` that we can reference from all other namespaces... But I don't think we're allowed to install these resources in the default control plane (yet).